crates/nu-command/src/filesystem/cp.rs

```diff
@@ -352,17 +352,32 @@ fn copy_file(src: PathBuf, dst: PathBuf, span: Span) -> Value {
             Value::String { val: msg, span }
         }
         Err(e) => {
-            let message = format!("copy file {src:?} failed: {e}");
-
+            let message_src = format!("copy file {src:?} failed: {e}");
+            let message_dst = format!("copy file {dst:?} failed: {e}");
```

Comment:
We should make clear that we refer to the destination

```suggestion
            let message_src = format!("copying file '{src_display}' failed: {e}", src.display());
            let message_dst = format!("copying to destination '{dst_display}' failed: {e}", dst.display());
```

(Little nitpick: we should probably format the output with quotes and escapes that you can directly paste in nushell. `Debug` will output `"` which support C style escapes and might break with windows paths)